HOW_TO_CREATE_A_PLAN.md

```diff
@@ -0,0 +1,45 @@
+## How to create a plan - Contributor version
+
+### Preamble 
+If you’re new to working with Expensify, it’s important to know the philosophy behind the design of Expensify’s products and the processes that wrap around them.
+- Problems are identified and defined before solutions are proposed
+- Features are not added to satisfy personal preferences
+- Features are added thoughtfully and minimally
+
+It’s also important to be familiar with our existing documentation including our [CONTRIBUTING.md](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md) and [README.md](https://github.com/Expensify/App/blob/main/README.md)
+
+## Step 1: Define the problem
+What is the problem that needs solving? 
+- Be broad
+- Define who experiences the problem
+- Does this affect the customer experience? How so? 
+- How many customers would be affected by this?
+- Define why it is a problem
+- Resist the temptation to reverse-engineer a Solution into a matching Problem.  (ie. Problem: We don’t have a car. Solution: Buy a car) 
```

Comment:
I am just wondering, the direction we should avoid is Solution -> Problem, should the bad example be something like: 

Solution: I want a car.
Problem: We do not have a car.

But this is just coming to my mind given the order of the problem/solution issue. Let me know what you think.

Reply:
I think you're right cuz it's "reverse-engineer a Solution" which means you would start with the solution to get the problem.. right?  If so, can you make that update?

Reply:
I like it 👍